kodilla-testing/src/test/java/com/kodilla/testing/collection/CollectionTestSuite.java

```diff
@@ -0,0 +1,62 @@
+package com.kodilla.testing.collection;
+
+import org.junit.*;
+import java.util.*;
+
+public class CollectionTestSuite {
+    static int counter = 1;
+
+    @Before
+    public void before(){
+        System.out.println("Now... Test Case " + counter + " begins");
+    }
+
+    @After
+    public void after() {
+        System.out.println("Now... Test Case " + counter + " ends");
+        counter++;
+    }
+
+    @Test
+    public void testOddNumbersExterminatorEmptyList(){
+        //Given
+        ArrayList<Integer> numbers = new ArrayList<Integer>();
+        OddNumbersExterminator oddNumbersExterminator = new OddNumbersExterminator();
+        //When
+        numbers = oddNumbersExterminator.exterminate(numbers);
+        //Then
+        boolean areThereAnyOdds = false;
+        for(Integer currentNumber : numbers){
+            if(currentNumber % 2 != 0){
+                areThereAnyOdds = true;
+            }
+        }
+        Assert.assertFalse(areThereAnyOdds);
+    }
```

Comment:
Zbedne komplikownie kodu. Znacznie latwiej to zapisac:

for(Integer currentNumber : numbers){
Assert.assertFalse(currentNumber % 2 != 0);
}

Reply:
Thank you for your valuable remarks. I have done the changes and commited them already.